### ISSUE TITLE
Feature request: unsuggested filter

### ISSUE THREAD

**Issue by JohnnyKing94** [translate/pootle#6667]
I'd like to see a new filter in Pootle and that is the **unsuggested** one.
Basically i ask to all my contributors to continue the translation with the untranslated filter, but sometimes you find strings already suggested by other users. It is not very efficient to have that in the untranslated filter 😄 

So basically i propose 2 more filter withe the same scope:

1. the pure unsuggested filter
2. and the combo of untranslated+unsuggested filter, it may be called `Untranslated (without suggestions)`

But if you want to extend the combination to all the filters in Pootle (eg. Incomplete+Unsuggested or Translated+Unsuggested etc), then it is better to have a new menu filter.

Basically, for me, i'll go with the first idea

**Comment by yakusha** [comment]
I'd like to see this feature being implemented soon.

I work on an open source translation project as a volunteer. There are more than 1.000 lines waiting being translated. I have finished half of them but its really hard to find which ones I didn't suggest a translation.

So, I need a new filter that I can find Untranslated+Unsuggested lines easily.

**Comment by sanecito** [comment]
Also came in here to chime for an unsuggested filter. The FOSS that I contribute to just had a batch add of roughly 1,300 strings and it's extremely tedious to find strings that don't have suggestions yet in a set of 1,300.
